scripts/core_scripts.lua

```diff
@@ -137,3 +165,150 @@ openspace.invertBooleanProperty = function(propertyIdentifier)
     end
     openspace.setPropertyValueSingle(propertyIdentifier, not value)
 end
+
+openspace.fadeIn = function(identifier, fadeTime, endScript)
+    -- Set default values for optional arguments
+    if endScript == nil then
+        endScript = ""
+    end
+
+    if fadeTime == nil then
+        fadeTime = openspace.getPropertyValue("OpenSpaceEngine.FadeDuration")
+    end
+
+    local enabledProperty = identifier .. ".Enabled"
+    local fadeProperty = identifier .. ".Fade"
+
+    -- Assume that we need to enable the node(s) as we fade it/them in
+    local isEnabled = false
+
+    -- Check if the identifier is a single node or a regex for several nodes
+    local hasTag, _ = identifier:find('{')
+    local hasWild, _ = identifier:find('*')
+
+    if hasTag ~= nil or hasWild ~= nil then
+        -- Regex, several nodes
+        local enabledPropertyList = openspace.getProperty(enabledProperty)
+        if next(enabledPropertyList) == nil then
+            -- List is empty, no matches found
+            openspace.printError(
```

Comment:
I believe there is a way to raise an error from Lua that will cause the script to finish executing;  basically the equivalent of a `throw` in Lua

Reply:
Sounds like a future problem.  I'll go ahead and merge